relay_test.go

```diff
@@ -565,6 +565,12 @@ func TestRelayRejectsDuringClose(t *testing.T) {
 		require.Error(t, err, "Expect call to fail after relay is shutdown")
 		assert.Contains(t, err.Error(), "incoming connection is not active")
 		close(block)
+		wg.Wait()
```

Comment:
If a `WaitGroup` is `Done`, but nobody is `Wait`ing, is it `Done`?

<img src="http://previews.123rf.com/images/yayayoy/yayayoy1205/yayayoy120500007/13629388-Meditating-emoticon-Stock-Vector-smiley-face-emoticon.jpg"  width=100>